README.md

```diff
@@ -63,6 +63,42 @@ RKE2_URL=https://myserver:6443 RKE2_TOKEN=XXX ./install.sh
 
 We provide a simple automated way to install RKE2 on AWS via terraform scripts, this method requires terraform to be installed and access to AWS cloud, to get started please checkout the [rke2-build](https://github.com/rancher/rke2-build) repo.
 
+## CentOS/RHEL 7 RPM Installation
+
+Signed RPM's are published for RKE2 within the `rpm.rancher.io` RPM repository. In order to use the RPM repository, on a CentOS 7 or RHEL 7 system, run the following bash snippet:
+
+```bash
+cat << EOF > /etc/yum.repos.d/rancher-rke2-1-18-latest.repo
+[rancher-rke2-common-latest]
+name=Rancher RKE2 Common Latest
+baseurl=https://rpm.rancher.io/rke2/latest/common/centos/7/noarch
+enabled=1
+gpgcheck=1
+gpgkey=https://rpm.rancher.io/public.key
+
+[rancher-rke2-1-18-latest]
+name=Rancher RKE2 1.18 Latest
+baseurl=https://rpm.rancher.io/rke2/latest/1.18/centos/7/x86_64
+enabled=1
+gpgcheck=1
+gpgkey=https://rpm.rancher.io/public.key
+EOF
+```
+
+After this, you can either run:
+
+```bash
+yum -y install rke2-server
+```
+
```

Comment:
Run yum makecache to download yum repository data to cache

Reply:
`yum makecache` is only necessary to pull the latest changes from the repository if there is a new RPM released between the first time you run `yum install` and later, otherwise yum will automatically build the metadata cache the first time you run `yum install`

Reply:
FYI, by default the repo metadata is cached for 6 hours. yum makecache (or yum clean all) should only be necessary in development or testing workflows where we're updating packages more frequently then that.